src/core/handle.ml

```diff
@@ -38,15 +38,16 @@ let _ =
 
 (** Representation of a yet unchecked proof. The structure is initialized when
     the proof mode is entered, and its finalizer is called when the proof mode
-    is exited (i.e., when a terminator like “qed” is used).  Note that tactics
+    is exited (i.e., when a terminator like “end” is used).  Note that tactics
     do not work on this structure directly,  although they act on the contents
     of its [pdata_p_state] field. *)
 type proof_data =
   { pdata_stmt_pos : Pos.popt (* Position of the proof's statement.  *)
   ; pdata_p_state  : Proof.t  (* Initial proof state for the proof.  *)
   ; pdata_tactics  : p_tactic list (* Tactics for the proof.         *)
   ; pdata_finalize : sig_state -> Proof.t -> sig_state (* Finalizer. *)
-  ; pdata_term_pos : Pos.popt (* Position of the proof's terminator. *) }
+  ; pdata_term_pos : Pos.popt (* Position of the proof's terminator. *)
```

Comment:
If this is indeed the position of the proof end, shouldn't we give it a better name like pdata_end_pos?

Reply:
fixed in 6ab82ed